backend/app/controllers/books_controller.rb

```diff
@@ -5,7 +5,9 @@ class BooksController < ApplicationController
   before_action :authorize_request
 
   def index
-    render(json: BookSerializer.new(books, meta_pagination(books, { params: { current_user: current_user } })),
+    render(json: BookSerializer.new(books.map do |book|
+                                      book
+                                    end, meta_pagination(books, { params: { current_user: current_user } })),
```

Comment:
can u put this aligned to render, the spacing is kinda weird lol or if u can u can also delegate this to a function so u can just pass it to the BookSerializer

def book_ref
  @book_ref ||= books.map { |book| book }
end